src/Promotions/WelcomeBanner/resources/js/app/components/Sections/RightContentSection.tsx

```diff
@@ -16,26 +17,29 @@ export default function RightContentSection({assets}: RightContentSectionProps)
             <h2>{__('Spotlight on awesome features', 'give')}</h2>
             <div className={'givewp-welcome-banner-right-content__media-container'}>
                 <SpotLight
-                    title={__('Design mode', 'give')}
+                    title={
+                        <>
+                            {__('Event tickets', 'give')} <span className={'givewp-beta-icon'}>BETA</span>
+                        </>
+                    }
                     description={__(
-                        'See exactly what your form looks like for potential donors using the “Design” tab of the builder. Changes are visible immediately.',
+                        'Easily connect your events to your donation form. To enable this go to Settings > General > New feature and enable event tickets',
                         'give'
                     )}
                 >
-                    <VideoPlayer src={`${assets}/design-mode.mp4`} fallbackImage={`${assets}/design-mode.min.png`} />
+                    <VideoPlayer
+                        src={`${assets}/event-tickets.mp4`}
```

Comment:
@jdghinson is going to replace this to use a more generic donor name for the example lol